<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputWnd</class>
 <widget class="QWidget" name="OutputWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>输出</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="m_TabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_AllInfoPage">
      <attribute name="title">
       <string>总览</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="m_OpenInfoPage">
      <attribute name="title">
       <string>解析</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="m_SaveInfoPage">
      <attribute name="title">
       <string>保存</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="m_AppInfoPage">
      <attribute name="title">
       <string>校验</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
